mmaction/datasets/pipelines/augmentations.py

```diff
@@ -639,6 +640,9 @@ def __init__(self,
                  area_range=(0.08, 1.0),
                  aspect_ratio_range=(3 / 4, 4 / 3),
                  lazy=False):
+        warnings.warn('"RandomResizedCrop" is deprecated since it is provided '
```

Comment:
"RandomResizedCrop" is deprecated and will be removed in later versions. It is currently not used in MMAction2.

Reply:
Emmm, actually I mean RandomScale is deperacated. Please check my last PR.

Reply:
RandomResizedCrop is OK.

Reply:
= =, my fault